src/main/webapp/help/ArtifactoryInstaller/help-serverId.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Server ID of your JFrog instance, which configured in Manage Jenkins | Configure System.
```

Comment:
...which configured in Manage Jenkins | Configure System....
-->
...which is configured in 'Manage Jenkins | Configure System'....